### PR TITLE
brltty: 5.4 -> 5.5

### PR DESCRIPTION
**Pull request by bramd**
###### Motivation for this change

New upstream version.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/brltty/default.nix [hunk 1]
```diff
@@ -4,14 +4,12 @@ assert alsaSupport -> alsaLib != null;
...
-  name = "brltty-5.4";
+  name = "brltty-5.5";
   
   src = fetchurl {
     url = "http://brltty.com/archive/${name}.tar.gz";
-    sha256 = "1993brxa76yf7z3ckax0bbmqv6jp8vjwxp19h425v4gpm0m17k7l";
+    sha256 = "0slrqanwj9cm7ql0rpb296xq676zrc1sjyr13lh5lygp4b8qfpci";
   };
-
-  patches = [ ./systemd.patch ];
```
Comment (Mic92) [inline pkgs/tools/misc/brltty/default.nix]: why was that removed?

Comment (bramd) [inline pkgs/tools/misc/brltty/default.nix]: The patch has been applied upstream. You are right, I removed the file as well.
